cassiopeia/simulator/UniformLeafSubsampler.py

```diff
@@ -62,14 +60,24 @@ def subsample_leaves(
         character states, meta data, and the dissimilarity map for the sampled
         cells only.
 
+        Has the option to keep the single edge leading from root in the
+        induced subtree, if it exists. This edge is often used to represent the
+        time that the root lives before any divisions occur in the phyologeny,
+        and is useful in instances where the branch lengths are critical, like
+        simulating ground truth phylogenies or estimating branch lengths.
+
         Args:
             tree: The CassiopeiaTree for which to subsample leaves
-            collapse_source: The source node from which to collapse
-                unifurcations
+            keep_root_unary_edge: Whether or not to collapse the single edge
+                leading from the root in the subsample, if it exists
 
         Returns:
             A new CassiopeiaTree that is the induced subtree on a sample of the
-            leaves in the given tree.
+            leaves in the given tree
+
+        Raises:
+            LeafSubsamplerError if the sample size is <= 0, or larger than the
+            number of leaves in the tree
```

Comment:
you'll have to indent the second line of the `Returns` section and the `Raises` section.